File: src/main/java/ttd/batch/Bookmark.java
package ttd.batch;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.ArrayList;
import java.util.Collection;
import java.util.Date;

/**
	* @author <a href="mailto:josh@joshlong.com">Josh Long</a>
	*/
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Bookmark {

		private Long bookmarkId;
		private String extended, description, meta, hash, href, publishKey;
		private Collection<String> tags = new ArrayList<>();
		private Date time;
}

File: src/main/kotlin/ttd/site/generator/Bookmark.kt
package ttd.site.generator

import java.util.*

data class Bookmark(
    val bookmarkId: Long,
    val extended: String,
    val description: String,
    val meta: String,
    val hash: String,
    val href: String,
    val publishKey: String,
    val tags: Collection<String>,
    val time: Date
)